adapter-cqld4/src/main/java/io/nosqlbench/adapter/cqld4/optionhelpers/OptionHelpers.java

```diff
@@ -196,15 +198,13 @@ private void addModifiers() {
                 Optional<Integer> maxRemote = Optional.ofNullable(matcher.group("rmax")).map(Integer::valueOf);
                 Optional<Integer> rqRemote = Optional.ofNullable(matcher.group("rrq")).map(Integer::valueOf);
                 if (coreRemote.isPresent() && maxRemote.isPresent() && !coreRemote.get().equals(maxRemote.get())) {
-                    throw new RuntimeException("In CQL Java driver 4, rcore and rmax connections have been reduced to a single value." +
-                        " You have two different values in (" + spec + "). If you make them the same, you can continue to use the 'pooling' helper option." +
-                        " Otherwise, set the driver options directly according to driver 4 docs.");
+                    throw new RuntimeException(
+                            STR."In CQL Java driver 4, rcore and rmax connections have been reduced to a single value. You have two different values in (\{spec}). If you make them the same, you can continue to use the 'pooling' helper option. Otherwise, set the driver options directly according to driver 4 docs.");
```

Comment:
Cases like this may benefit from the block readability of the """ approach more, or possibly combined with it. The long wraps are not a format we want to have in the code base in general. This comment applies as well to all changes where block structured code is converted to a long line wrap.

Reply:
This comment was attached to the "before" view of the code and it should have been for the "after" view with the long unbroken string form.